src/controller/CommissionerDiscoveryController.cpp

```diff
@@ -440,6 +442,9 @@ void CommissionerDiscoveryController::InternalHandleContentAppPasscodeResponse()
             }
             CharSpan rotatingIdSpan(rotatingIdBuffer, 2 * rotatingIdLength);
 
+            // Store rotating ID as tempAccountIdentifier to use in AccountLogin::Login payload later.
+            mTempAccountIdentifier = rotatingIdSpan;
```

Comment:
This is use-after-return, no?

Reply:
Yes, good catch. Will refactor in line with chrisdecenzo comments, but will keep this in mind when dealing with spans. Thanks.

Reply:
Code removed, fixed.